gorjana/flows/products/product-set-up-sop.md

```diff
@@ -1,67 +1,79 @@
+2. Link products to bins in all facilities/locations for inventory synchronization.
+
+### Steps in Shopify
+1. Set up the corresponding products in Shopify and ensure no duplicate SKUs exist, regardless of product status (draft/active).
+2. Link all products to all facilities/locations in Shopify to enable inventory synchronization.
+
+### Steps to Sync Products in HotWax
+1. Run the [Import Product job](https://job-manager.hotwax.io/product) manually from the Products section of the Job Manager. This imports all the products set up in Shopify.
+2. NetSuite product details are synced into HotWax every 6 hours via scheduled jobs.
+3. For immediate synchronization, follow these manual steps:
+   - Run the [HC_SC_GenerateProductCSV](https://4054670.app.netsuite.com/app/common/scripting/scriptrecord.nl?id=6432) SuiteScript.
+   - Run the [HC_SC_UploadProductCSV](https://4054670.app.netsuite.com/app/common/scripting/scriptrecord.nl?id=6433) SuiteScript.
+   - Run the [Import Product Identification Job](https://job-manager.hotwax.io/product) job from the Products section of the Job Manager.
+   - Run the [HC_SC_GenerateProdSalesDescriptionCSV](https://4054670.app.netsuite.com/app/common/scripting/scriptrecord.nl?id=6489) SuiteScript.
+   - Run the [HC_SC_UploadProdSalesDescriptionCSV](https://4054670.app.netsuite.com/app/common/scripting/scriptrecord.nl?id=6490) SuiteScript.
+   - Run the [Import NetSuite Sales Description Job](https://job-manager.hotwax.io/product) job from the Products section of the Job Manager.
+
+4. For a limited number of products, manually add the NetSuite Internal Product ID in HotWax:
+   - Navigate to **OMS → Hamburger Menu → PIM →** [Products](https://gorjana-oms.hotwax.io/commerce/control/FindProduct).
+   - Search for the SKU, open the product page, and click the `+` symbol in the **Identifications** section.
+   - Select "NetSuite Product Internal ID," enter the description, and save.
+
+> **NOTE**
+> - NetSuite resets inventory in HotWax daily at 1 a.m.
+> - HotWax syncs inventory with Shopify at 3 a.m. (Inventory will sync only to linked locations in Shopify.)
+> - Thus, invenotry for new products will be in sync between the three systems only the day after the product was set up.
+
+
+
+## 2. Setting Up Kit Products
+1. Import the components and the kit product itself by following the steps in **Setting Up New Products in HotWax**.
+2. Export the kit parent-child association from NetSuite using the [HC_MR_ExportedKITProductCSV](https://4054670.app.netsuite.com/app/common/scripting/scriptrecord.nl?id=6461) SuiteScript.
+3. Wait up to 10 minutes for the kit component details to import into HotWax.
+
+
+## 3. Editing Existing Products
+
+### Syncing Changes Made in Shopify
+1. Make any required changes in Shopify.
+2. A scheduled job in HotWax imports product updates every hour.
+3. For immediate synchronization, run the **Import Product Updates** job from the Products section of the Job Manager.
```

Comment:
I'd say for immediate change user should go through EXIM create up date MDM right?

Reply:
That’s a valid point. Using EXIM for create/update is just as fast as running the job from the Job Manager and gives the user control over what’s being imported.

However, when products are updated in bulk, running the job would be convenient. I suggest we present both alternatives in the documentation and let users decide based on their specific needs. 

I’ll edit accordingly, if you agree.

Reply:
As discussed in person, it is better to import using the EXIM as running the job imports all product updates since the last run time (and any sale/inventory change on a product also counts as an update). Thus, the same can be mentioned to the user.